JEMS-CSE416/JEMS
# Create map modal

To test:

**Setup:**
```
git fetch
git checkout create_map_modal
cd client
npm install
npm start
```

**Testing**
1. Go to browser
2. Click Get Started Button or navigate to `/home`
3. You should see a create map modal with all of the appropriate fields
